ovos_phal_connectivity_events/__init__.py

```diff
@@ -36,41 +38,71 @@ class ConnectivityEvents(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus=bus, name="ovos-PHAL-plugin-connectivity-events", config=config)
         self.sleep_time = 60
+        self.stopping = Event()
         self.state = ConnectivityState.UNKNOWN
         self.bus.on("ovos.PHAL.internet_check", self.handle_check)
-        self.bus.emit(Message("ovos.PHAL.internet_check"))
-
-    def update_state(self, state, message):
-        if state == ConnectivityState.FULL:
-            # has internet
-            if self.state <= ConnectivityState.NONE:
+        if not self.config.get('disable_scheduled_checks'):
+            self.bus.emit(Message("ovos.PHAL.internet_check"))
+
+    def run(self):
+        if self.config.get('disable_scheduled_checks'):
+            self.stopping.wait()
+        while not self.stopping.wait(self.sleep_time):
+            self.handle_check(None)
+
+    def shutdown(self):
+        PHALPlugin.shutdown(self)
```

Comment:
why not use super? this works fine just interested if theres some corner case this is handling